src/locales/fi.json

```diff
@@ -38,9 +38,9 @@
       "noMatches": "Ei hakua vastaavia osumia"
     },
     "sidebar": {
-      "harbors": "Venepaikat",
+      "harborList": "Venepaikat",
```

Comment:
Some side bar translations broken
<img width="278" alt="Screenshot 2020-06-01 at 16 30 36" src="https://user-images.githubusercontent.com/1407958/83413978-63015b80-a425-11ea-850e-a6d14c4bf587.png">

Reply:
Good catch, that's embarrassing. Fixed